### PR TITLE
[Spring] Fix the incorrect split of byoc commands and args.

### PR DESCRIPTION
**Pull request by LGDoor**
---
Fixed a regression bug introduced in the previous refactoring that the commands and arguments aren't split correctly.

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
spring cloud

File: src/spring-cloud/azext_spring_cloud/_params.py [hunk 1]
```diff
@@ -364,9 +364,9 @@ def prepare_logs_argument(c):
             c.argument(
                 'registry_password', help='The password of the container registry.', arg_group='Custom Container')
             c.argument(
-                'container_command', help='The command of the container image.', nargs='*', arg_group='Custom Container')
+                'container_command', help='The command of the container image.', arg_group='Custom Container')
             c.argument(
-                'container_args', help='The arguments of the container image.', nargs='*', arg_group='Custom Container')
+                'container_args', help='The arguments of the container image.', arg_group='Custom Container')
```
Comment (zhoxing-ms) [inline src/spring-cloud/azext_spring_cloud/_params.py:369]: I would like to ask whether the change will cause breaking change to users?

**Comment by zhoxing-ms** [comment]
Could you add some tests for those change?

**Comment by zhoxing-ms** [comment]
Please add description of those change to the `HISTORY.rst` file. By the way, do you need to release a new version of extension for this PR? If so, please also modify the `setup.py` file

File: src/spring-cloud/azext_spring_cloud/_params.py (hunk 1 above)
Comment (LGDoor) [reply src/spring-cloud/azext_spring_cloud/_params.py:369]: No. Actually the correct type is `string`. It's changed to the array accidentally in a previous refactoring.

Comment (zhoxing-ms) [reply src/spring-cloud/azext_spring_cloud/_params.py:369]: OK

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**